cinema/models.py

```diff
@@ -35,12 +39,24 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> str:
+    _, extension = os.path.splitext(filename)
+    return os.path.join(
+        "uploads/images",
+        f"{slugify(instance.title)}-{uuid.uuid4()}{extension}",
+    )
```

Comment:
Read how os.path.join works

```suggestion
    return os.path.join(
        "uploads", "images",
        f"{slugify(instance.title)}-{uuid.uuid4()}{extension}",
    )
```